dll/win32/iphlpapi/ifenum_reactos.c

```diff
@@ -149,7 +149,9 @@ BOOL hasArp( HANDLE tcpFile, TDIEntityID *arp_maybe ) {
                               NULL );
     if( !NT_SUCCESS(status) ) return FALSE;
 
-    return (type & AT_ARP);
+    /* This was previously treated incorrectly as a single bitmapped flag.
+     * But this is not correct because AT_ARP is 0x280. */
```

Comment:
A suggestion for the comment:
```suggestion
    /* AT_ARP corresponds to an individual TDI entity type */
```

Reply:
@HBelusca, My comment will not make sense after this commit, but I included it to help explain what was happening in hopes that it would speed the commit. I was thinking that it would be quickly deleted and/or changed and your suggestion certainly look like a good substitute. Again, maybe the committer can implement this for us. Thanks.